src/coreclr/src/inc/clrinternal.idl

```diff
@@ -128,30 +128,6 @@ typedef VOID (__stdcall *PTLS_CALLBACK_FUNCTION)(PVOID);
 ]
 interface IExecutionEngine : IUnknown
```

Comment:
My ultimate goal is to delete the whole IExecutionEngine indirection. The TLS slots are big enough piece to deserve dedicated PR.